corehq/apps/linked_domain/tasks.py

```diff
@@ -108,7 +108,7 @@ def release(self, models, linked_domains, build_apps=False):
 
     def send_email(self):
         subject = _("Linked project release complete.")
-        if self.errors_by_domain:
+        if self.errors_by_domain['html']:
```

Comment:
Could this check `_get_error_domain_count` instead? I'd like to keep the details of these structures internal to the `ReleaseManager` class.